bigbluebutton-html5/imports/ui/components/whiteboard/service.js

```diff
@@ -326,13 +330,14 @@ const changeCurrentSlide = (s) => {
   makeCall("changeCurrentSlide", s);
 }
 
-const getShapes = (whiteboardId, curPageId, intl) => {
+const getShapes = (whiteboardId, curPageId, intl, limit) => {
   const annotations =  Annotations.find(
     {
       whiteboardId,
     },
     {
       fields: { annotationInfo: 1, userId: 1, },
+      limit,
```

Comment:
I noticed one caveat with this, if a poll result is published, the annotation is not being parsed if we are at the max shapes limit.
I would prefer to allow the poll annotation to go above the limit